DataFormats/Reconstruction/src/TrackParametrizationWithError.cxx

```diff
@@ -1247,8 +1244,8 @@ namespace o2::track
 {
 #if !defined(GPUCA_GPUCODE) || defined(GPUCA_GPUCODE_DEVICE) // FIXME: DR: WORKAROUND to avoid CUDA bug creating host symbols for device code.
 template class TrackParametrizationWithError<float>;
-#endif
-#ifndef GPUCA_GPUCODE
+#ifndef __OPENCL__
```

Comment:
This is to avoid to produce symbols for `trackparcov<double>` which clashes with SinCos() implementation for OpenCL: https://github.com/AliceO2Group/AliceO2/blob/bfa4f1e320833dcf7861d75803b26a9592fcce49/Common/MathUtils/include/MathUtils/detail/trigonometric.h#L112

Reply:
Here I am not 100% convinced. Do we need the double version on the GPU?
If not I would leave this out for now. Not sure e.g. if all the CAMath functions work in double for GPU.

Reply:
No, in ITS we don't need TrackParCov<double>, point taken for the `CAMath` functions.
I put it back behind guards.